packages/fake/lib/plugin.js

```diff
@@ -24,33 +24,33 @@ export default class FakePlugin extends BasePlugin {
     },
   };
 
-  static fakeRoute (req, res) {
+  static fakeRoute (_req, res) {
```

Comment:
what is the purpose of adding underscore prefixes?

Reply:
this just indicates, ignore if value is unused

Reply:
I would rather keep it as is. Also, in the overridden classes it is not mandatory to have the same argument names for method arguments. Only order matters